tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/AspNetCoreTestFixture.cs

```diff
@@ -259,6 +258,27 @@ private async Task EnsureServerStarted(bool sendHealthCheck)
             }
         }
 
+        private bool IsPortListening(int port)
+        {
+            try
+            {
+                using (var client = new TcpClient())
+                {
+                    var task = client.ConnectAsync("127.0.0.1", port);
```

Comment:
I assume `localhost` will always resolve to `127.0.0.1`, even in docker containers etc 🤔 I guess we'll find out - as long as the tests work, I think we're ok! 😄

Reply:
That's an interesting topic. Initially, I used localhost, but in some tests, the ConnectAsync call would fail in local tests but not in others.